src/qminer/qminer_storage.h

```diff
@@ -1177,6 +1179,36 @@ class TStorePbBlob : public TStore, public TToaster {
     /// Initialize field location flags
     void InitDataFlags();
 
+    /// Do we have a primary field
```

Comment:
This code is shared with TStoreImpl, would be nice to move it to separate class so both store implementations can share it.